### PR TITLE
psychiatrist job is now slightly less terrible

### PR DESCRIPTION
**Pull request by fluffe9911**
In a effort to make psy slightly less terrible I gave them a locker with some random pill bottles and dice, a paper bin clipboard and pen to right down how his subjects feeellll and a book binder encase the they want to be a self published author and stuff that is all.

🆑 
rscadd: The Psychiatrist now spawns with some little goodies!
/:cl:

### CODE REVIEW

File: _maps/yogstation/map_files/YogStation/YogStation.dmm [hunk 1]
```diff
@@ -35139,6 +35142,20 @@
 	icon_state = "pipe11-1";
 	dir = 6
 	},
+/obj/structure/closet/secure_closet{
+	name = "psychiatrist locker";
+	req_access_txt = "5"
+	},
+/obj/item/storage/pill_bottle/happiness,
+/obj/item/storage/pill_bottle/floorpill,
```
Comment (ThatLing) [inline _maps/yogstation/map_files/YogStation/YogStation.dmm:35150]: Why?

**Comment by Jc1469** [comment]
just give them a russian revolver that has 6 real bullets in it

File: _maps/yogstation/map_files/YogStation/YogStation.dmm (hunk 1 above)
Comment (fluffe9911) [reply _maps/yogstation/map_files/YogStation/YogStation.dmm:35150]: cause imagining the psy giving people who come in a completely random pill is funny

Comment (ThatLing) [reply _maps/yogstation/map_files/YogStation/YogStation.dmm:35150]: Floor pills were meant to be maint only and I am not okay with this

**Pull request merged**